openslides/motions/views.py

```diff
@@ -138,7 +138,7 @@ def create(self, request, *args, **kwargs):
                     del request.data[key]
 
         # Check permission to send comment data.
-        if not has_perm(request.user, 'motions.can_see_and_manage_comments'):
+        if not has_perm(request.user, 'motions.can_manage_comments'):
```

Comment:
The user should have to have both perms.

Reply:
Done.